apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependences
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import handler from './src/handler';
+
+window.onerror = handler( apiFetch );
```

Comment:
Can we `addEventListener` instead of assigning to `onerror`?

```suggestion
window.addEventListener( 'error', handler );
```

Reply:
👍🏻

Reply:
That's a good point, thanks for pointing it out. We never know if any plugins will eventually assign to this handler and assigning globally like this is definitely smelly, listening to the event on `window` is a much safer approach.